### ISSUE TITLE
Themes: Add alt attributes from Showcase Data

### ISSUE THREAD

**Issue by davidakennedy** [Automattic/wp-calypso#34091]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/themes
2. View source to see no alt attribute on theme screenshots.

#### What I expected

I expected there to be an alt attribute because I've added one in the theme showcase install. It looks like we have some code in place, but not sure where it's coming from. See: https://github.com/Automattic/wp-calypso/blob/d8d2ba47ec5a1bed7fd7146acd9fa4ed980d3fd9/client/components/theme/index.jsx#L223

#### What happened instead

No alt attribute was added, meaning screen reader users can't know what the themes look like.

#### Context / Source
Reported through a customer: #user-report.

**Comment by sdixon194** [comment]
If possible, notify 2123869-zen when fixed

**Comment by bluefuton** [comment]
The `Theme` does indeed look for a `description` field on the theme object, but it isn't there because the API endpoint doesn't provide it.

I've worked up an API patch to remedy that: D29741-code.

I think we'll also need a small Calypso PR to strip HTML entities out of the retrieved description.

**Comment by bluefuton** [comment]
🚢D29741-code has shipped and we're now including the description in the `alt` attribute. 

<img width="937" alt="Screen Shot 2019-06-25 at 11 34 55" src="https://user-images.githubusercontent.com/17325/60058659-5d8f9180-973d-11e9-87ef-c2d4bda13a9f.png">
